src/components/Observations/plot/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import strings from 'src/strings';
+import DetailsPage from 'src/components/Observations/common/DetailsPage';
+
+export default function ObservationMonitoringPlotDetails(): JSX.Element {
+  const { plantingSiteId, observationId, plantingZoneId, monitoringPlotId } = useParams<{
+    plantingSiteId: string;
+    observationId: string;
+    plantingZoneId: string;
+    monitoringPlotId: string;
+  }>();
+
+  const urlSite = APP_PATHS.OBSERVATIONS_SITE.replace(':plantingSiteId', plantingSiteId?.toString());
+
+  const urlDetails = `/${observationId}`;
+
+  const urlZone = `/${plantingZoneId}`;
```

Comment:
this is simpler now, the bread crumbs component will build the full url (now that `hierarchical` is set to true)